superset/config.py

```diff
@@ -383,6 +383,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
     # for report with type 'report' still send with email and slack message with
     # screenshot and link
     "ALERTS_ATTACH_REPORTS": True,
+    "FORCE_SSL": False,
```

Comment:
Can you add documentation about what this flag does?

Reply:
Yeah! where should I add the documentation?

Reply:
inline, similarly to the other feature flags would be great, thanks!